project.json

```diff
@@ -13,5 +13,9 @@
     ],
     "icon_path": "preview.png",
     "engine": "o3de",
-    "external_subdirectories": []
+    "external_subdirectories": [],
+    "gem_names": [
+        "XR", // added via call to enable-gem
+        "OpenXRVk" // added via call to enable-gem
```

Comment:
Are these comments actually added by the `o3de.bat` script?  I just worry about normalizing this since it's non-standard for Json to have comments like this.

Reply:
+1